modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/systemDataSets.spec.ts

```diff
@@ -23,10 +27,24 @@ export const test = mergeTests(
 	loginTest()
 );
 
+const dataSetERCs: string[] = [];
+
+test.afterEach(async ({dataSetManagerApiHelpers}) => {
+	for (const erc of dataSetERCs) {
+		await dataSetManagerApiHelpers.deleteDataSet({
+			erc,
+		});
+	}
+});
```

Comment:
@thektan this is the mechanism I mentioned in https://github.com/liferay-frontend/liferay-portal/pull/4650#issuecomment-2618412254. It did require adding erc to markup. It's easier when we create objects with API, but we are fully testing UI in this test.